### PR TITLE
Add game logic

### PR DESCRIPTION
**Pull request by JoscaWij**
- create player (appearance with img)
- create virus as opstical (appearance with img)
- place obstacle randomly on canvas 
- create pill-object (appearance with img) that makes player 10s immune to virus
- create new objects after a certain amount of time with array and for.Each method 
- implement  collusion-detaction [(location obstacle = location player) = kill player] and add new objects to the detaction 
- if player collides with obstacle the player stops
-  incrase speed of player with each new obstacle
- Add dungeon background 
- update reset button (erases obstacles)
- create component: popup in case of the death of the player / game over [not responsive, WIP]
- create component: endscreen - player survives the game [not yet responsive, and not implemented, WIP] [implementation: after a certain amount of obstacles or after collection of a certain amount of pills?]

**deployment:** https://mj-s-invasion.github.io/storybook-game/?path=/story/components-game--basic

### CODE REVIEW

**Comment by JoscaWij** [comment]
game: 
image

pop-up:
image

endscreen:
image

File: src/components/game/game.js [hunk 1]
```diff
@@ -76,22 +148,77 @@ const movePlayer = (player, timePassed) => {
...
+          return false;
+        }
+        return true;
+      });
+
+      obstacles.find((obstacle) => {
+        let distance = calcDistance(player, obstacle);
+        if (distance < 1 && !virusImmune) {
+          player.speed = 0;
+          aliveTime.innerText = 0;
```
Comment (manuelsanchez2) [inline src/components/game/game.js:194]: love it!!!

File: src/components/popUpWindow/popUpWindow.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  });
+  const imgContainer = createElement("div", { className: "imgContainer" });
+
+  const popUp = createElement("div", { className: "popUp" });
+  const textGameOVer = createElement("p", { innerText: "GAME OVER" });
+  const textInf = createElement("p", { innerText: "YOU GOT INFECTED" });
+
+  popUpWindow__container.append(popUp);
+  popUpWindow__container.append(imgContainer);
+  popUp.append(textGameOVer);
```
Comment (manuelpapa) [inline src/components/popUpWindow/popUpWindow.js:20]:
```suggestion
  popUp.append(textGameOver);
```

**Review by manuelsanchez2 — APPROVED** [review]

File: src/components/popUpWindow/popUpWindow.js (hunk 2 above)
Comment (manuelpapa) [reply src/components/popUpWindow/popUpWindow.js:20]: hard to find even one problem within your code :D.

**Review by manuelpapa — COMMENTED** [review]
Awesome, great job. Has BEM, consistency, logic and style ;)

**Pull request merged**